radix-engine-common/src/crypto/hash.rs

```diff
@@ -75,7 +76,7 @@ pub fn hash<T: AsRef<[u8]>>(data: T) -> Hash {
 //========
 
 /// Represents an error when parsing hash.
-#[derive(Debug, Clone, PartialEq, Eq)]
+#[derive(Debug, Clone, PartialEq, Eq, ScryptoSbor)]
```

Comment:
Can we change this to `Sbor` rather than `ScryptoSbor` so hash is not dependent on scrypto objects.

Reply:
Done